projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/shipping-address/address-form/address-form.component.html

```diff
@@ -14,7 +14,6 @@
                   [items]="titles"
                   bindLabel="name"
                   bindValue="code"
-                  placeholder="Select Title..."
```

Comment:
why exactly was this placeholder attribute deleted?

Reply:
After add option "none", this select is always selected so placeholder will be not displayed anymore